components/Objectives/ChallengeList/ListItem.js

```diff
@@ -61,7 +62,7 @@ export default function ChallengeListItemComponent({
               : `/objective/${objectiveId}/challenge-detail/${challengeId}`
           }
         >
-          Edit ✏️
+          Edit <EditIcon disabled={false} />
```

Comment:
Same here: I don't think you need the `disabled` prop always set to `false`.

Reply:
Same as above 😊